Packs/CommonTypes/IndicatorFields/incidentfield-description.json

```diff
@@ -18,7 +18,7 @@
     "breachScript": "",
     "editForm": true,
     "description": "",
-    "associatedToAll": false,
+    "associatedToAll": true,
```

Comment:
You changed this field in the mutex PR, lets choose one.a

Reply:
Its "true" in both, because this PR is forked from the mutex one which should be merged first. It's on purpose